src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2114,25 +2114,75 @@ namespace UnitarySystems {
                     this->m_NoLoadAirFlowRateRatio = min(NoLoadCoolingAirFlowRateRatio, NoLoadHeatingAirFlowRateRatio);
                 }
             } else {
-                if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed ||
-                    this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingAirToAirVariableSpeed) {
-                    if (this->m_CoolCoilExists && this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed) {
+                if ((this->m_CoolCoilExists || this->m_HeatCoilExists) && this->m_noLoadLowSpeed) {
+                    if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed ||
+                        this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingWaterToAirHPVSEquationFit) {
                         Real64 MaxSpeedFlowRate =
                             state.dataVariableSpeedCoils->VarSpeedCoil(this->m_CoolingCoilIndex)
                                 .MSRatedAirVolFlowRate(state.dataVariableSpeedCoils->VarSpeedCoil(this->m_CoolingCoilIndex).NumOfSpeeds);
                         if (MaxSpeedFlowRate > 0.0) {
                             NoLoadCoolingAirFlowRateRatio =
                                 state.dataVariableSpeedCoils->VarSpeedCoil(this->m_CoolingCoilIndex).MSRatedAirVolFlowRate(1) / MaxSpeedFlowRate;
+                        } else {
+                            // I think all these little IFs need an else in case these inputs are autosized proportional to number of speeds/stages
+                            NoLoadCoolingAirFlowRateRatio = state.dataVariableSpeedCoils->VarSpeedCoil(this->m_CoolingCoilIndex).NumOfSpeeds;
```

Comment:
This needs to be 1 / NumOfSpeeds.

Reply:
I think I had to add these `} else { to cover autosized inputs where MaxSpeedFlowRate = -99999 here before the coil is sized. So yeah, this does need to change.

Reply:
These are the speed air flows, and they are allowed to be 0. So this does look OK.

    Coil:Cooling:DX:VariableSpeed,
    N19, \field Speed 1 Reference Unit Rated Air Flow Rate
        \units m3/s
        \type real
        \minimum 0
        \required-field

    N91, \field Speed 10 Reference Unit Rated Air Flow Rate
        \units m3/s
        \type real
        \minimum 0